index.html

```diff
@@ -227,7 +227,20 @@ <h3>
         </ul>
       </div>
     </section>
-
+    <section id="contact">
+      <footer>
+          <h2>Contact me</h2>
+          <p id="para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="#" method="post">
+            <input class="name" type="text" name="user-name" placeholder="Write Your Name Here">
```

Comment:
- [ ] As stated in the project requirements: Please ensure that the name input field accepts up to 30 chars. You can do this by adding the maxlength="30" attribute

Reply:
> * [ ]  As stated in the project requirements: Please ensure that the name input field accepts up to 30 chars. You can do this by adding the maxlength="30" attribute

- [ ] Great Work